### DISCUSSION TITLE
Version 0.2.9.9-adoption `<Adoption>`

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/ElephantEngine#33] [Announcements]
## Version 0.2.9.9-adoption `<Adoption>`

* 17 Dec 2021
* fRelease for Kerbal Space Program [KSP 1.12.x]

## Adoption by @zer0Kerbal

* update pass
* automation pass
* organizational structure pass

### Localization

* Create
* Localization/
  * <en-us.cfg>
  * [readme.md] v2.1.1.0
  * [quickstart] v1.0.1.0
* updates #12 - Localization - Master
* closes #13 - Localization - English (United States)] <en-us.cfg>

### Documentation

* change Changelog.cfg --> changelog.md
* update release version

### Status

* Issues
  * closes #3 - Repo
  * closes #4 - Adoption
  * closes #5 - Social
  * closes #6 - Release
  * closes #7 - PreRelease
  * closes #8 - Legal MumboJumbo
  * closes #9 - Update documentation
  * closes #10 - Release
  * closes #11 - Social
* Pull Requests
  * #29 - 0.2.9.9 adoption
  * #30 - Update and rename ElephantEngine-README.txt to readme.md - contributed by zer0Kerbal

## What's Changed
* Update and rename ElephantEngine-README.txt to readme.md by @zer0Kerbal in https://github.com/zer0Kerbal/ElephantEngine/pull/30
* 0.2.9.9 adoption by @zer0Kerbal in https://github.com/zer0Kerbal/ElephantEngine/pull/29
* Version 0.2.9.9-adoption `<Adoption>` by @zer0Kerbal in https://github.com/zer0Kerbal/ElephantEngine/pull/32

## New Contributors
* @zer0Kerbal made their first contribution in https://github.com/zer0Kerbal/ElephantEngine/pull/30

**Full Changelog**: https://github.com/zer0Kerbal/ElephantEngine/compare/0.2.0.0...0.2.9.9-adoption

<hr /><em>This discussion was created from the release <a href='https://github.com/zer0Kerbal/ElephantEngine/releases/tag/0.2.9.9-adoption'>Version 0.2.9.9-adoption `<Adoption>`</a>.</em>

**Comment by AtomikkuSan** [comment]
*Proceeds to Laugh Evilly*
